cellbase-app/app/azure/arm/webservers/azuredeploy.json

```diff
@@ -0,0 +1,432 @@
+            "package_reboot_if_required: true",
+            "runcmd:",
+            "- curl -fsSL https://get.docker.com/ | sh",
+            "- curl -fsSL https://get.docker.com/gpg | sudo apt-key add -",
+            "[concat('- ', variables('logAnalyticsAgent'))]",
+            "[concat('- docker run --name=cellbase -d -p 8080:8080 -p 8443:8443 --restart=always -e CELLBASE_DATABASES_MONGODB_HOST=',string(parameters('mongoHosts')),' -e  CELLBASE_DATABASES_MONGODB_USER=',string(parameters('mongoUsername')),' -e  CELLBASE_DATABASES_MONGODB_PASSWORD=',string(parameters('mongoPassword')),'  -e CELLBASE_DATABASES_MONGODB_OPTIONS_AUTHENTICATION_DATABASE=admin  -e CELLBASE_DATABASES_MONGODB_OPTIONS_REPLICA_SET=  -e  CELLBASE_DATABASES_MONGODB_OPTIONS_READ_PREFERENCE=nearest  -e CELLBASE_DATABASES_MONGODB_OPTIONS_SSL_ENABLED=true  ', parameters('cellbaseContainerImage'))]",
+            ""
+        ],
+        "cloud-init-string": "[replace(replace(replace(string(variables('cloud-init')),'\",\"','\n'),'[\"',''),'\"]','')]",
+        "customData": "[base64(variables('cloud-init-string'))]"
+     },
+    "resources": [
+      
+        {
+            "comments": "The LB configured to send traffic to the web nodes in the VMSS. It monitors the health endpoint of OpenCGA removing unhealthy nodes from the LB",
+            "type": "Microsoft.Network/loadBalancers",
+            "name": "[variables('loadBalancerName')]",
+            "location": "[resourceGroup().location]",
+            "apiVersion": "2016-03-30",
+            "dependsOn": [
+             
+              
+            ],
+            "properties": {
+                "loadBalancingRules": [
+                    
+                    {
+                        "name": "cellbaseHttp",
+                        "properties": {
+                            "frontendIPConfiguration": {
+                                "id": "[variables('privateFrontEndIPConfigID')]"
+                            },
+                            "backendAddressPool": {
+                                "id": "[concat(variables('lbID'),'/backendAddressPools/VMSSBackendPool')]"
+                            },
+                            "protocol": "Tcp",
+                            "frontendPort": 8080,
+                            "backendPort": 8080,
+                            "enableFloatingIP": false,
+                            "idleTimeoutInMinutes": 5,
+                            "probe": {
+                                "id": "[concat(variables('lbID'),'/probes/cellbaseHttpProbe')]"
+                            }
+                        }
+                    }
+                ],
+                "probes": [
+                    {
+                        "name": "cellbaseHttpProbe",
+                        "properties": {
+                            "protocol": "Http",
+                            "port": 8080,
+                            "requestPath": "/opencga/webservices/rest/v1/meta/about",
```

Comment:
Does this probe location work on cellbase? Thought it was an opencga specific thing.

Reply:
It's gives a 200 if the app is up, which is enough for now.